src/types.rs

```diff
@@ -0,0 +1,22 @@
+use core::fmt;
+
+#![allow(non_camel_case_types)]
```

Comment:
Travis can't compile this PR right now because this must be the first line of the file.

Reply:
Opss, did this after the fact and missed this.

Reply:
Fixed it https://github.com/rust-bitcoin/rust-secp256k1/compare/f42cfcb..becb35a